python/llm/example/CPU/HF-Transformers-AutoModels/Model/yuan2/README.md

```diff
@@ -0,0 +1,74 @@
+# Yuan2
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Yuan2 models. For illustration purposes, we utilize the [IEITYuan/Yuan2-2B-hf](https://huggingface.co/IEITYuan/Yuan2-2B-hf) as a reference Yuan2 model.
+
+## 0. Requirements
+To run these examples with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+In addition, you need to modify some files in Yuan2-2B-hf folder, since Flash attention dependency is for CUDA usage and cannot be installed on Intel CPUs. To manually turn it off, please refer to [this issue](https://github.com/IEIT-Yuan/Yuan-2.0/issues/92).
+
+1. Modify 'use_flash_attention' to false in config.json; Comment out lines 35 and 36 in yuan_hf_model.py; 
```

Comment:
To make it convenient for users, please provide a new `yuan_hf_mode.py` file with these modifications, as we did in falcon example https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/falcon